README.md

```diff
@@ -101,6 +101,10 @@ Advanced feature. Use this to opt in / out prettier on various language ids. Res
 Use parser `graphql` for given language ids.
 Use with care.
 
+#### prettier.ignorePath (default: .prettierignore)
+Supply the path to an ignore file such as `.gitignore` or `.prettierignore`.
+Files which match will not be formatted. Set to `null` to not read ignore files. Restart required.
```

Comment:
Little nitpick: `Restart required.`, of what exactly? The plugin, the editor, the window?

Reply:
All of them in fact.

We have same annotation to other settings ({LANG}Enabled)

Reply:
Allright, than we can keep this.